homeassistant/components/bbb_gpio/__init__.py

```diff
@@ -4,13 +4,15 @@
 from Adafruit_BBIO import GPIO  # pylint: disable=import-error
 
 from homeassistant.const import EVENT_HOMEASSISTANT_START, EVENT_HOMEASSISTANT_STOP
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.typing import ConfigType
 
 DOMAIN = "bbb_gpio"
 
 _LOGGER = logging.getLogger(__name__)
 
 
-def setup(hass, config):
+def setup(hass: HomeAssistant, config: ConfigType) -> bool:
```

Comment:
This falls under [ADR19](https://github.com/home-assistant/architecture/blob/master/adr/0019-GPIO.md), should I remove this change, or is it OK to include it in "bulk cleanup"?

Reply:
Nah its ok.